pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -310,11 +353,11 @@ func (vm *windows) transferFiles() error {
 		// of file that we want to transfer, WMCO team should cut a newer version of operator.
 		// The files are tightly coupled with the operator
 		// TODO: Remove this when we do in place upgrades
-		out, err := vm.Run("Test-Path "+dest+"\\"+filepath.Base(src), true)
+		fileExists, err := vm.FileExists(dest + "\\" + filepath.Base(src))
```

Comment:
Ideally you introduce `vm.FileExists()` in a separate commit, use it in all existing places where it is open coded.

Reply:
There seems to be only 1 place where it is open coded (right here). Is that enough to warrant a separate commit?

Reply:
It is a good practice to follow. Some projects would require separate PRs :-)